src/components/Tricots.js

```diff
@@ -3,50 +3,65 @@ import {Header, List, Image} from "semantic-ui-react";
 import {Helmet} from "react-helmet";
 import SingleTrikot from "./SingleTrikot";
 import * as maillotActions from '../actions/maillotActions'
+import * as riderStageConnectionsActions from "../actions/riderStageConnectionsActions";
 import store from '../store'
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'store'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)